src/xx-apk

```diff
@@ -46,8 +46,8 @@ setup() {
 
   cp /etc/apk/repositories "$apk_dir/"
 
-  # riscv only supported in edge atm
-  if [ "$TARGETARCH" = "riscv64" ]; then
+  # riscv supported since alpine 3.20
+  if [ "$TARGETARCH" = "riscv64" ] && ! supportRiscV; then
```

Comment:
Should we switch these users to 3.20 instead? Like:

3.19 - 3.20
3.20 - 3.20
3.21 - 3.21
edge - edge

Reply:
Yes I think that's fair